packages/block-library/src/embed/variations.js

```diff
@@ -360,6 +361,14 @@ const variations = [
 		patterns: [ /^https?:\/\/(www\.)?wolframcloud\.com\/obj\/.+/i ],
 		attributes: { providerNameSlug: 'wolfram-cloud', responsive: true },
 	},
+	{
+		name: 'bluesky',
+		title: 'Bluesky',
+		icon: embedBlueskyIcon,
+		description: __( 'Embed a Bluesky post.' ),
+		patterns: [ /^https?:\/\/(www\.)?bsky\.app\/obj\/.+/i ],
+		attributes: { providerNameSlug: 'bluesky', responsive: true },
```

Comment:
```suggestion
		attributes: { providerNameSlug: 'bluesky' },
```

When the `responsive` property is `true`, a control appears in the block sidebar that determines whether to keep the aspect ratio when the browser is resized.

![responsive-setting](https://github.com/WordPress/gutenberg/assets/54422211/50fbbfba-89a8-4204-85dc-96ae97d302fb)

Bluesky changes the aspect ratio depending on the content, so I think this setting is probably unnecessary.